lib/unplug/predicates/app_config_in.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Unplug.Predicates.AppConfigIn do
+  @moduledoc """
+  Given an application and a key, execute the plug if the configured value
+  matches the expected value.
```

Comment:
For the `In` style predicates, I would just update the moduledoc to make it clear that the configured value is in an enumerable of values. Something like `if the value is in the the provided list of values.`